.eslintignore

```diff
@@ -1,3 +1,4 @@
 coverage/*
 dist/
 node_modules/
+src/postcss.config.js
```

Comment:
nit: Add a newline to the end of the file.

Reply:
fine